src/db/mysql.schema

```diff
@@ -264,6 +264,22 @@ CREATE TABLE `WEBSOCKET_MESSAGE_FUZZ` (
   KEY `SYS_FK_10181` (`MESSAGE_ID`,`CHANNEL_ID`),
   CONSTRAINT `SYS_FK_10181` FOREIGN KEY (`MESSAGE_ID`, `CHANNEL_ID`) REFERENCES `WEBSOCKET_MESSAGE` (`MESSAGE_ID`, `CHANNEL_ID`)
 ) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+
+DROP TABLE IF EXISTS `STRUCTURE`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `STRUCTURE` (
+  `STRUCTUREID` bigint(20) NOT NULL AUTO_INCREMENT,
+  `SESSIONID` bigint(20) NOT NULL,
+  `PARENTID` bigint(20) NOT NULL,
+  `HISTORYID` int(11) DEFAULT NULL,
+  `NAME` text NOT NULL,
+  `NAMEHASH` bigint(20) NOT NULL,
+  `URL` text NOT NULL,
+  `METHOD` varchar(10) NOT NULL,
+  PRIMARY KEY (`STRUCTUREID`)
+) ENGINE=InnoDB AUTO_INCREMENT=17487 DEFAULT CHARSET=utf8mb4;
```

Comment:
- The `AUTO_INCREMENT` should be removed. The `CHARSET` should be the same as the other tables.
 - The query `structure.ps.createtable` (in `src/db/mysql.properties`) should also be fixed.